keras_nlp/models/xlm_roberta/xlm_roberta_tasks.py

```diff
@@ -115,3 +121,62 @@ def from_config(cls, config):
         if "backbone" in config:
             config["backbone"] = keras.layers.deserialize(config["backbone"])
         return cls(**config)
+
+    @classproperty
+    def presets(cls):
+        return copy.deepcopy(backbone_presets)
+
+    @classmethod
+    @format_docstring(names=", ".join(backbone_presets))
+    def from_preset(
+        cls,
+        preset,
+        load_weights=True,
+        **kwargs,
+    ):
+        """Create a classification model from a preset architecture and weights.
+
+        Args:
+            preset: string. Must be one of {{names}}.
+            load_weights: Whether to load pre-trained weights into model.
+                Defaults to `True`.
+
+        Examples:
+        ```python
+        input_data = {
+            "token_ids": tf.random.uniform(
+                shape=(1, 12), dtype=tf.int64, maxval=model.vocabulary_size
+            ),
+            "padding_mask": tf.constant(
+                [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0], shape=(1, 12)
+            ),
+        }
+
+        # Load backbone architecture and weights from preset
+        classifier = XLMRobertaClassifier.from_preset(
```

Comment:
This may be a change we need to make elsewhere too, but we should make sure to use the full symbol here.

keras_nlp.models.XLMRobertaClassifier.from_preset